### PR TITLE
Nept 229: Fix insufficient access check on views in common features

### PR DESCRIPTION
**Pull request by gillesdeudon**

### CODE REVIEW

File: profiles/common/modules/features/e_library/e_library_core/e_library_core.views_default.inc [hunk 1]
```diff
@@ -223,6 +224,7 @@ function e_library_core_views_default_views() {
   /* Display: Page */
   $handler = $view->new_display('page', 'Page', 'page');
   $handler->display->display_options['defaults']['arguments'] = FALSE;
+  $handler->display->display_options['path'] = 'e_library';
```
Comment (dxvargas) [inline profiles/common/modules/features/e_library/e_library_core/e_library_core.views_default.inc]: Was this path intended to stay? Seems like a change made for testing proposes.

File: profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module [hunk 2]
```diff
@@ -450,11 +450,25 @@ function nexteuropa_multilingual_path_update($path) {
...
+    || !is_numeric($parsed_source[1])) {
+    return;
+  }
+  $nid = $parsed_source[1];
+  $query = db_select('node', 'n')->condition('n.nid', $nid);
+  $query->addField('n', 'type', 'node_type');
+  $node_type = $query->execute()->fetchField();
+
+  if ($node_type) {
+    $translation_mechanism = variable_get('language_content_type_' . $node_type, 0);
```
Comment (dxvargas) [inline profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module]: This change has nothing to do with current task, it shouldn't be in this PR.

File: profiles/common/modules/features/e_library/e_library_core/e_library_core.views_default.inc (hunk 1 above)
Comment (gillesdeudon) [inline profiles/common/modules/features/e_library/e_library_core/e_library_core.views_default.inc]: No, No. It is normal. If you active only this module, the Views Ui raises an error. it is for fixing this error.

**Comment by dxvargas** [comment]
👍

**Comment by ademarco** [comment]
👍

**Pull request merged**